### ISSUE TITLE
add ExAC non TCGA  to annotations

### ISSUE THREAD

**Issue by brentp** [arq5x/gemini#588]
ExAC now provides a site VCF that excludes TCGA samples. There is also a non-psych sites VCF.
We could probably merge all 3 in the provenance and add, e.g. AC_ntcga=...;AC_npsyc=...;
so that we don't add 2 extra huge files to the downloads.

**Comment by davemcg** [comment]
Yes please! I've been using this in my VEP annotations. I was thinking I could just use Gemini's ExAC, then I wondered "wait, what version are they using?" I suggest the nonTCGA ExAC numbers become the default for Gemini.
